docusaurus/docs/iOS/uikit/components/thread-list.md

```diff
@@ -0,0 +1,316 @@
+let threadListController = ChatClient.shared.threadListController(query: threadListQuery)
+let eventsController = ChatClient.shared.eventsController()
+let threadListVC = ChatThreadListVC(
+    threadListController: threadListController,
+    eventsController: eventsController
+)
+
+// Present the thread list VC
+let navigationThreadListVC = UINavigationController(rootViewController: threadListVC)
+navigationController?.pushViewController(navigationThreadListVC, animated: true)
+```
+
+You can present the Thread List in a `UINavigationController` or in a `UITabBarController` whatever fits your app design. The `ChatThreadListVC` has two required dependencies: a `ThreadListController` and an `EventsController`.
+
+## Thread List Query
+
+The `ThreadListQuery` is responsible to configure the list of threads that will be displayed in the `ChatThreadListController`. These are the available parameters:
+
+- `watch`: A boolean indicating whether to watch for changes in the thread list or not.
+- `limit`: The amount of threads fetched per page. The default is 20.
+- `replyLimit`: The amount of replies fetched per thread. The default is 3.
+- `participantLimit`: The amount of participants fetched per thread. The default is 10.
+- `next`: The pagination token from the previous response to fetch the next page.
+
+All the parameters are customizable and you can change them according to your needs. The default values are a good compromise between performance and user experience.
+
+If you are using the `ChatThreadListVC` component, you don't need to worry about the `next` parameter, since pagination is handled for you. If not, you can use the `next` parameter from the previous response to fetch the next page of threads.
+
+## UI Customization
+
+You can customize the Thread List UI by overriding the `ChatThreadListVC` component and the `ChatThreadListItemView`. In the `ChatThreadListVC` you can change, for example, the header and state views, where in the `ChatThreadListItemView` you can change how each thread is displayed.
+
+### Thread List Navigation Header
+
+The navigation header of the thread list can be configured the same way you would configure it on a `UIViewController` object if it is part of a `UINavigationController`. You can change the `title` and the `navigationItem` properties. Here is an example:
+
+```swift
+class CustomThreadListVC: ChatThreadListVC {
+    
+     override open func setUpAppearance() {
+        super.setUpAppearance()
+        title = "Threads"
+    }
+}
+```
+
+### Thread List Header Banner View
+
+The `ChatThreadListHeaderBannerView` is the view that is displayed by default as a `tableHeaderView` in the `ChatThreadListVC`. This view is shown whenever the user is on the Thread List and new threads are available to fetch. New threads are not added automatically to the Thread List on propose so that the user can manually decide to fetch them whenever he is ready and has read all the current unread threads.
```

Comment:
`propose` > `purpose`

Reply:
He is, should be correct 🤔 Im talking about one user

Reply:
I think what he meant is to make the sentence gender neutral, not that it's grammatically incorrect.

Reply:
Ah ok 👍

Reply:
Done ✅